<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>500</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"># border-image:url(&quot;images/page1.png&quot;)</string>
    </property>
    <widget class="QWidget" name="verticalWidget" native="true">
     <property name="geometry">
      <rect>
       <x>-14</x>
       <y>-10</y>
       <width>985</width>
       <height>320</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0;</string>
     </property>
     <layout class="QVBoxLayout" name="chartContainer">
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>374</width>
       <height>200</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="trafficContainer">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="trafficContent">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <widget class="QProgressBar" name="recv_send_ratio">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>50</y>
              <width>131</width>
              <height>75</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>1</pointsize>
              <fontweight>Thin</fontweight>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LayoutDirection::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QProgressBar {
    border-radius: 37px;
    background-color: #ff5151;
    text-align: center;
    color: black;
}

QProgressBar::chunk {
    background-color: #fff700;
    border-top-left-radius: 37px;
    border-bottom-left-radius: 37px;
}</string>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>45</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QLabel" name="recv_kbs">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>70</y>
              <width>80</width>
              <height>20</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255,255);</string>
            </property>
            <property name="text">
             <string>0.0 KB/s</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="send_kbs">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>70</y>
              <width>80</width>
              <height>20</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255,255);</string>
            </property>
            <property name="text">
             <string>0.0 KB/s</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
           <widget class="Line" name="line_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>80</x>
              <y>-10</y>
              <width>187</width>
              <height>20</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #a3a3a3;</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>2</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
           <widget class="QLabel" name="sum_kbs">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>20</y>
              <width>131</width>
              <height>20</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>18</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255,255);</string>
            </property>
            <property name="text">
             <string>0.0 KB/s</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
           <widget class="QFrame" name="right_radius_mask">
            <property name="geometry">
             <rect>
              <x>204</x>
              <y>50</y>
              <width>37</width>
              <height>75</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <stylestrategy>PreferAntialias</stylestrategy>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
           </widget>
           <widget class="QFrame" name="left_radius_mask">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>50</y>
              <width>37</width>
              <height>75</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <stylestrategy>PreferAntialias</stylestrategy>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
           </widget>
           <widget class="QFrame" name="send_vector">
            <property name="geometry">
             <rect>
              <x>249</x>
              <y>85</y>
              <width>10</width>
              <height>7</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
           </widget>
           <widget class="QFrame" name="recv_vector">
            <property name="geometry">
             <rect>
              <x>89</x>
              <y>86</y>
              <width>11</width>
              <height>7</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
           </widget>
           <widget class="QLabel" name="send_kbs_2">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>90</y>
              <width>80</width>
              <height>20</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #c3c3c3;</string>
            </property>
            <property name="text">
             <string>0.0 KB</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
            </property>
           </widget>
           <widget class="QLabel" name="recv_kbs_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>90</y>
              <width>80</width>
              <height>20</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #c3c3c3;</string>
            </property>
            <property name="text">
             <string>0.0 KB</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTop|Qt::AlignmentFlag::AlignTrailing</set>
            </property>
           </widget>
           <widget class="Line" name="line_3">
            <property name="geometry">
             <rect>
              <x>347</x>
              <y>42</y>
              <width>3</width>
              <height>88</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #a3a3a3;</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Plain</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>374</x>
       <y>300</y>
       <width>586</width>
       <height>200</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="NetworkContainer">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="NetworkContent">
        <property name="styleSheet">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QFrame" name="groupBox">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <widget class="QLabel" name="WAN">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>66</y>
              <width>41</width>
              <height>16</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255,255);</string>
            </property>
            <property name="text">
             <string>전 세계</string>
            </property>
           </widget>
           <widget class="QLabel" name="LAN">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>96</y>
              <width>41</width>
              <height>16</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255,255);</string>
            </property>
            <property name="text">
             <string>내 지역</string>
            </property>
           </widget>
           <widget class="QProgressBar" name="LAN_BAR">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>100</y>
              <width>400</width>
              <height>10</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>1</pointsize>
              <fontweight>Thin</fontweight>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QProgressBar {
    border-radius: 5px;
    background-color: #ff5151;
    text-align: center;
    color: black;
}

QProgressBar::chunk {
    background-color: #fff700;
    border-top-left-radius: 5px;
    border-bottom-left-radius: 5px;
}</string>
            </property>
            <property name="value">
             <number>45</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="Line" name="line">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>-10</y>
              <width>293</width>
              <height>20</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #a3a3a3;</string>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>2</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
           <widget class="QProgressBar" name="WAN_BAR">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>70</y>
              <width>400</width>
              <height>10</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>1</pointsize>
              <fontweight>Thin</fontweight>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QProgressBar {
    border-radius: 5px;
    background-color: #ff5151;
    text-align: center;
    color: black;
}

QProgressBar::chunk {
    background-color: #fff700;
    border-top-left-radius: 5px;
    border-bottom-left-radius: 5px;
}</string>
            </property>
            <property name="value">
             <number>45</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QLabel" name="WAN_LABEL">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>66</y>
              <width>80</width>
              <height>16</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #c3c3c3;</string>
            </property>
            <property name="text">
             <string>0.0 KB</string>
            </property>
           </widget>
           <widget class="QLabel" name="LAN_LABEL">
            <property name="geometry">
             <rect>
              <x>50</x>
              <y>96</y>
              <width>80</width>
              <height>16</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #c3c3c3;</string>
            </property>
            <property name="text">
             <string>0.0 KB</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>935</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
